HW0/P6/P6.txt

```diff
@@ -1 +1,86 @@
-# Your answers here
+Problem 6.
```

Comment:
Your graph for problem 6 looks like it has a lot of noise and doesn't give much insight into the trend.